Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -321,16 +247,139 @@ - (void)_stealFirstResponder;
 
 - (void)_showBugTitleCaptureAlert;
```

Comment:
@rhgills please take a look at this method and make sure it won't break iPad for you.